Automattic/wp-calypso
# Importers: Move importer configuration to a centralized location.

#### Changes proposed in this Pull Request

* Moves the only file in `lib/importers` (`url-validation`) to `lib/importer`
* Removes the individual importer constants in `state/imports/constants`
* Removes the config from individual importer components and centralizes it to `lib/importer/importer-config`.

#### Testing instructions



This PR shouldn't result in any visual difference to the flow.

* Visit `/import/yoursite.wordpress.com`. Verify that the list of importers is displayed the same, and in the same order.
* Click "Start Import" next to one of the listed importers. Confirm that you're taken to the file upload step (except for GoDaddy / Wix, where you'll be shown a URL input)
* Click "Cancel", verify that you are returned to the list of importers.
* Continue by selecting "Start Import" again. Confirm that you're able to complete an import process.
* Visit `/import/yoursite.wordpress.com?engine=wordpress`. Verify that you are taken directly to the WordPress importer.
* Visit `/import/yoursite.wordpress.com?engine=wix&from-site=yoursite.wixsites.com/site`. Verify that you are taken directly to the Wix importer and that a preview of your Wix site is displayed.
